### DISCUSSION TITLE
Unable to find enrollment profile url

### DISCUSSION THREAD

**Discussion by narendra-bhole** [micromdm/nanomdm#115] [Q&A] [answered]
Hello,

I'm unable to find out  nanomdm url for downloading enrollment profile. I tried using https://{nanomdm-server}/mdm. It's giving response as Bad Request. Please help me on this.

**Comment by jessepeterson** [comment, marked as answer]
Hello @narendra-bhole. As mentioned in the README it is out of scope for the NanoMDM project itself to host enrollment profiles. It is up to you to get them to your devices however you wish -- be that a web server of your own, emailed profiles, or what have you.

That said I might suggest taking a look at this project: https://github.com/korylprince/dep-webview-oidc
